Write documentation (by po) (#97)
GeneriekPublicatiePlatformWoo/registratie-component: docs/admin/Metadata/index.rst

.. admin_Metadata_index:

> .. _`audittrail` : ../Logging/index.hmtl
> .. _`Wet open overheid` : https://wetten.overheid.nl/BWBR0045754/
> .. _`Woo-index`: https://open.overheid.nl/
> .. _`DiWoo` : https://standaarden.overheid.nl/diwoo/metadata
> .. _`Woo informatiecategorieën` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
> .. _`Thema-indeling voor Officiële Publicaties (TOP-lijst)` : https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
> .. _`audittrail` : ../Logging/index.hmtl

Reviewer: we tend to put the link targets at the bottom of a file usually to increase readability of the code

additionally, you don't need to wrap the aliases in backticks for the link definitions :)

```suggestion
.. _Wet open overheid: https://wetten.overheid.nl/BWBR0045754/
.. _Woo-index: https://open.overheid.nl/
.. _DiWoo: https://standaarden.overheid.nl/diwoo/metadata
.. _Woo informatiecategorieën: https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_woo_informatiecategorieen
.. _Thema-indeling voor Officiële Publicaties (TOP-lijst): https://standaarden.overheid.nl/tooi/waardelijsten/work?work_uri=https%3A%2F%2Fidentifier.overheid.nl%2Ftooi%2Fset%2Fscw_toplijst
```

This also removes the internal links, for those we can use the `ref` syntax, e.g.:

```rst
:ref:`admin_logging_index`
```

> Metadata
> ============

Reviewer: ```suggestion
Metadata
========
```

Onder het menu-item "Metadata" en op het dashboard onder het kopje "Metadata" wordt toegang geboden tot het beheer van:

* Informatiecategorieën
* Thema's

> Door herop te klikken wordt het desbetreffende beheerscherm geopend.

Reviewer: ```suggestion
Door hierop te klikken wordt het desbetreffende beheerscherm geopend.
```

> Informatiecategorieën
> ------------------------

Reviewer: ```suggestion
Informatiecategorieën
---------------------
```

> Toelichting
> ~~~~~~~~~~~~~

Reviewer: ```suggestion
Toelichting
~~~~~~~~~~~
```

De `Wet open overheid`_ beschrijft 17 *informatiecategorieën*. Documenten die onder minstens één van deze categorieën vallen, moeten *actief* openbaar gemaakt worden. Dat wil zeggen dat een bestuursorgaan deze online moet publiceren, ook wanneer daar niet door een burger expliciet om gevraagd is in een *Woo-verzoek*. 

Daarnaast vraagt de wet om ook documenten openbaar te maken die buiten deze *informatiecategorieën* vallen. Hiervoor geldt geen "harde" plicht, maar wel een *inspanningsverplichting*.

> Overigens gelden op de plicht tot openbaarmaking ook een aantal uitzonderingen (hoofdstuk 5 van de wet). Raadpleeg voor meer informatie over de wet de jurist en/of informatiebeheerder van uw eigen organisatie.

Reviewer: ```suggestion
Overigens gelden op de plicht tot openbaarmaking ook een aantal uitzonderingen (hoofdstuk 5 van de wet). Raadpleeg voor meer informatie over de wet de jurist en/of informatiebeheerder van je eigen organisatie.
```
recent developments show that information language (in user interfaces, documentation...) is preferred

> Voor de aansluiting op de landelijke `Woo-index`_ wordt ten zeerste geadviseerd om documenten te voorzien van metadata conform de `DiWoo`_-standaard. Deze standaard vereist omder meer dat in de metadata wordt vastgelegd op grond van welke *informatiecategorie(ën)* het document actief openbaar is gemaakt.

Reviewer: ```suggestion
Voor de aansluiting op de landelijke `Woo-index`_ wordt ten zeerste geadviseerd om documenten te voorzien van metadata conform de `DiWoo`_-standaard. Deze standaard vereist onder meer dat in de metadata wordt vastgelegd op grond van welke *informatiecategorie(ën)* het document actief openbaar is gemaakt.
```

> De 17 *informatiecategorieën* zijn opgenomen in de landelijke waardelijst `Woo informatiecatgeorieën`_. Deze lijst is al standaard opgenomen in het "Woo Publications"-component. Daarnaast bestaat de mogelijkheid, zoals we hieronder zullen beschrijven, om in het kader van de *inspanningsverplichting* hier voor de eigen organisatie *extra informatiecategorieën* aan toe te voegen.

Reviewer: ```suggestion
De 17 *informatiecategorieën* zijn opgenomen in de landelijke waardelijst `Woo informatiecatgeorieën`_. Deze lijst is al standaard opgenomen in het "Woo Publicaties"-component. Daarnaast bestaat de mogelijkheid, zoals we hieronder zullen beschrijven, om in het kader van de *inspanningsverplichting* hier voor de eigen organisatie *extra informatiecategorieën* aan toe te voegen.
```

> In het "WOO Publications"-component leggen we de *informatiecategorieën* vast op de *publicatie* (in ontwikkeling). Dezelfde *informatiecategorieën* gelden voor de daaraan gekoppelde *documenten*.

Reviewer: ```suggestion
In het "WOO Publicaties"-component leggen we de *informatiecategorieën* vast op de *publicatie* (in ontwikkeling). Dezelfde *informatiecategorieën* gelden voor de daaraan gekoppelde *documenten*.
```
we passen de NL vertalingen ook in de interface toe :)

> Beheerscherm
> ~~~~~~~~~~~~~~

Reviewer: ```suggestion
Beheerscherm
~~~~~~~~~~~~
```

> In het beheerscherm van de *informatiecatgeorieën* wordt een lijst getoond van alle *informatiecategorieën* die zijn opgeslagen in het "WOO Publications"-component. 

Reviewer: ```suggestion
In het beheerscherm van de *informatiecatgeorieën* wordt een lijst getoond van alle *informatiecategorieën* die zijn opgeslagen in het "WOO Publicaties"-component. 
```

Standaard staan de landelijke 17 verplichte *informatiecategorieën* hier al in. Zij zijn te herkennen aan de waarde "Waardelijst" in de kolom `Oorsprong` van de lijst. Zij kunnen niet gewijzigd worden.
Daarnaast bestaat er de mogelijkheid om voor de eigen organisatie *extra informatiecategorieën* toe te voegen. Deze zijn te herkennen aan de waarde "Zelf-toegevoegd item" in de kolom `Oorsprong` van de lijst. Deze kunnen indien nodig gewijzigd worden.

Op het beheerscherm zijn de volgende acties mogelijk:

> * Rechtboven zit een knop **"informatiecategorie toevoegen"** waarmee een *informatiecategorie* toegevoegd kan worden.

Reviewer: ```suggestion
* Rechtboven zit een knop **informatiecategorie toevoegen** waarmee een *informatiecategorie* toegevoegd kan worden.
```
quotes zijn niet nodig door de bold formatting, dat maakt de documentatie ook meer consistent met Open Zaak en Open Formulieren etc.

* Bovenaan zit een zoekveld met en knop **"Zoeken"** waarmee naar *informatiecategorieën* gezocht kan worden.
* Daaronder zit de mogelijkheid om **eenzelfde actie uit te voeren over meerdere informatiecategorieën**. Op dit moment wordt alleen de actie **"Geselecteerde informatiecategorieën verwijderen"** ondersteund. Merk op dat het mogelijk is om in de lijst één of meerdere *informatiecategorieën* aan te vinken.
* Onder de (bulk-)actie staat de lijst met *informatiecategorieën*. Door op de kolomtitels te klikken kan de lijst **alfabetisch of chronologisch geordend** worden. 
* Rechts naast de lijst bestaat de mogelijkheid om deze te **filteren op oorsprong**.
* Bij een *informatiecategorie* kan op de `naam` geklikt worden om **de details in te zien** en deze eventueel **te wijzigen**.

> * Bij een *informatiecategorie* kan op **"Show logs"** (op één na rechter kolom) geklikt worden om direct de `audittrail`_ in te zien.

Reviewer: self-reminder: vertaling bijwerken zodra die opgevoerd is

> * Bij een *informatiecategorie* kan op **"Show logs"** (op één na rechter kolom) geklikt worden om direct de `audittrail`_ in te zien.

Reviewer: ```suggestion
* Bij een *informatiecategorie* kan op **"Show logs"** (op één na rechter kolom) geklikt worden om direct de :ref:`audit trail <admin_logging_index>` in te zien.
```
-> dit refereert naar het label `admin_logging_index` op een andere plek in de docs, en maakt de linktitel "audit trail"

Wanneer bij een *informatiecategorie* op  de `naam` wordt geklikt, wordt een scherm geopend met nadere details.
Hierop zien we:

* **Alle gegevens**. Deze lichten we hieronder toe.
* Rechtsboven een knop **"Show logs"**. Deze toont de volledige `audittrail`_ van de *informatiecategorie*.

> * Rechtsboven een knop **"Geschiedenis"**. Deze toont de beheer-handelingen die vanuit de Admin-interface zijn uitgevoerd op de *informatiecategorie*.

Reviewer: ```suggestion
* Rechtsboven een knop **"Geschiedenis"**. Deze toont de beheer-handelingen die vanuit de beheerinterface zijn uitgevoerd op de *informatiecategorie*.
```
Voorkeur voor Nederlands?

> * Als de *informatiecategorie* tot de landelijke 17 behoort, linksonder een knop om het scherm te **sluiten** en teurg te keren naar de lijst.

Reviewer: ```suggestion
* Als de *informatiecategorie* tot de landelijke lijst behoort, linksonder een knop om het scherm te **sluiten** en terug te keren naar de lijst.
```
maakt het wat makkelijker als het aantal in de toekomst wijzigt? :)

* Als de *informatiecatgeorie* door de organisatie zelf is toegevoegd, linksonder de mogelijkheid om **wijzigingen op te slaan** (indien van toepassing). Er kan voor gekozen worden om na het opslaan direct een nieuwe *informatiecategorie* aan te maken of om direct de huidige *informatiecategorie* nogmaals te wijzigen.
* Rechtsonder de mogelijkheid om de *informatiecategorie* te **verwijderen**.

Op een informatiecategorie zijn de volgende gegevens beschikbaar. Op het scherm wordt verplichte velden **dikgedrukt** weergegeven.

> * ``Naam``. De naam van de *informatiecategorie*, bijvoorbeeld "convenant".
> * ``Naam meervoud``. De meervoudsvorm van de naam, bijvoorbeeld "convenanten".
> * ``Definitie``. De (mogelijk landelijke) definitie van de *informatiecategorie*.
> * ``UUID``. Een niet-muteerbaar, automatisch toegekend identificatie kenmerk.
> * ``Oorsprong``. Een niet-muteerbaar, automatisch toegekende aanduiding van op welke wijze de *informatiecategorie* is toegevoegd; via de landelijke waardelijst of zelf toegevoegd.

Reviewer: ```suggestion
Naam
    De naam van de *informatiecategorie*, bijvoorbeeld "convenant".

Naam meervoud
    De meervoudsvorm van de naam, bijvoorbeeld "convenanten".

Definitie
    De (mogelijk landelijke) definitie van de *informatiecategorie*.

UUID
    Een niet-wijzigbaar, automatisch toegekend identificatiekenmerk.

Oorsprong
    Een niet-wijzigbaar, automatisch toegekende aanduiding van op welke wijze de *informatiecategorie* is toegevoegd; via de landelijke waardelijst of zelf toegevoegd.
```

* ben niet zeker of deze deifnition list syntax per se beter is, laat ik aan jou om te bepalen
* muteerbaar -> wijzigbaar ivm. toegankelijk taalgebruik waarbij eenvoudigere woorden de voorkeur hebben.

> Thema's
> --------

Reviewer: ```suggestion
Thema's
-------
```

> Toelichting
> ~~~~~~~~~~~~~
> Om de vindbaarheid van openbare documenten te bevorderen ondersteunt de `DiWoo`_-standaard het toekennen van een of meerdere *thema's* aan openbare documenten. Hiervoor is een landelijke waardelijst gedefinieerd: `Thema-indeling voor Officiële Publicaties (TOP-lijst)`_. Op de `Woo-index`_ kan een burger zoeken naar openbare documenten, die aan een bepaald thema gekoppeld zijn.

Reviewer: ```suggestion
Toelichting
~~~~~~~~~~~

Om de vindbaarheid van openbare documenten te bevorderen ondersteunt de `DiWoo`_-standaard het toekennen van een of meerdere *thema's* aan openbare documenten. Hiervoor is een landelijke waardelijst gedefinieerd: `Thema-indeling voor Officiële Publicaties (TOP-lijst)`_. Op de `Woo-index`_ kan een burger zoeken naar openbare documenten, die aan een bepaald thema gekoppeld zijn.
```

> De landelijke thema-lijst is ook standaard ingelezen in en wordt ontsloten met het "Woo-publications"-component, zodat deze gebruikt kan worden bij het registreren en vindbaar maken van openbare documenten.

Reviewer: ```suggestion
De landelijke thema-lijst is ook standaard ingelezen in en wordt ontsloten met het "Woo-publicaties"-component, zodat deze gebruikt kan worden bij het registreren en vindbaar maken van openbare documenten.
```

> In het "WOO Publications"-component leggen we de *thema's* vast op de *publicatie* (in ontwikkeling). Dezelfde *thema's* gelden voor de daaraan gekoppelde *documenten*.

Reviewer: ```suggestion
In het "WOO Publicaties"-component leggen we de *thema's* vast op de *publicatie* (in ontwikkeling). Dezelfde *thema's* gelden voor de daaraan gekoppelde *documenten*.
```

De `DiWoo`_-standaard ondersteunt alleen het gebruik van de landelijke waardelijst `Thema-indeling voor Officiële Publicaties (TOP-lijst)`_. Ook op de `Woo-index`_ kan alleen op deze *thema's*  gezocht worden naar openbare documenten.

> Beheerscherm
> ~~~~~~~~~~~~~~

Reviewer: ```suggestion
Beheerscherm
~~~~~~~~~~~~
```

* Daaronder zit de mogelijkheid om **eenzelfde actie uit te voeren over meerdere informatiecategorieën**. Op dit moment wordt alleen de actie **"Geselecteerde thema's verwijderen"** ondersteund. Merk op dat het mogelijk is om in de lijst één of meerdere *thema's* aan te vinken.
* Onder de (bulk-)actie staat de lijst met *thema's*. 
* Bij een *thema* kan op de `naam` geklikt worden om **de details in te zien**.
* Bij een *thema* kan op **"Show logs"** (rechter kolom) geklikt worden om direct de `audittrail`_ in te zien.

Wanneer bij een *thema* op  de `naam` wordt geklikt, wordt een scherm geopend met nadere details.
Hierop zien we:

* **Alle gegevens**. Deze lichten we hieronder toe.
* Rechtsboven een knop **"Show logs"**. Deze toont de volledige `audittrail`_ van het *thema*.
* Rechtsboven een knop **"Geschiedenis"**. Deze toont de beheer-handelingen die vanuit de Admin-interface zijn uitgevoerd op het *thema*.
* Linksonder een knop om het scherm te **sluiten** en teurg te keren naar de lijst.
* Rechtsonder de mogelijkheid om het *thema* te **verwijderen**.

Op een *thema* zijn de volgende gegevens beschikbaar.

> * ``UUID``. Een niet-muteerbaar, automatisch toegekend identificatie kenmerk.

Reviewer: ```suggestion
* ``UUID``. Een niet-wijzigbaar, automatisch toegekend identificatie kenmerk.
```

* ``Naam``. De naam van het *thema*, bijvoorbeeld "cultuur en recreatie".
* ``position``. (In ontwikkeling)

> * ``ref node id``. (In ontwikkeling)

Reviewer: zijn technische velden voor de boomstructuur, heeft de beheerder verder weinig mee van doen. Kan je afdoen als "systeemvelden"
